### DISCUSSION TITLE
v2ray有没有当成代理服务器服务端使用

### DISCUSSION THREAD

**Discussion by onlineyh** [v2fly/v2ray-core#2679] [Q&A]
比如说：我在内网中部署v2ray服务端，不用于翻墙。用户使用客户端认证成功后连接访问我内网的资源！

**Comment by AkinoKaede** [comment]
I think VPN will suit your needs better.

**Comment by onlineyh** [comment]
vpn一般需要管理员权限 添加路由，域控下操作比较麻烦。这个直接使用代理 普通用户都可以操作，也没有改变什么。使用很方便！！！

**Comment by onlineyh** [comment]
这种做成网页订阅号的模式，有改变直接修改base64，对方就更新了。使用确实比vpn、squid方便。而且也轻量级无需安装专门的客户端工具

**Comment by garryforreg** [comment]
完全可以，一直在用。 局域网内任意一台nas或者主机搭建v2ray服务端,把端口从路由上映射出来,客户端连接上之后就等于在内网了

**Reply by slePPyMe** [reply]
我也想做类似的事情，在软路由设置了常规的vmess inbound，防火墙开发了对应的端口。
客户端参照这个链接的思路：https://www.v2ex.com/t/920102，实现了开着小火箭同时访问内网和翻墙。
但现在内网的部分网页打不开，例如homeassistant，可能是由于使用了ws长连接？没有查到相关资料。别的网页可以打开。 你有遇到过吗？烦请赐教。

**Comment by onlineyh** [comment]
有没有具体实施可以参考的文档，我目前使用v2ray透明代理，https://guide.v2fly.org/app/tproxy.html 但是发现不行，因为他自动dns解析发现是国外的ip直接使用代理，国内ip使用目前的网络就已经出去了。我不知道如何操作？烦请赐教！
